### PR TITLE
byobu

### PR DESCRIPTION
**Pull request by setharnold**

### CODE REVIEW

File: 2019/7xxx/CVE-2019-7306.json [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+                ]
             }
         ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "refsource": "CONFIRM",
+                "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-7306"
+            },
```
Comment (iamleot) [inline 2019/7xxx/CVE-2019-7306.json:86]: Please remove this self-reference.

File: 2019/7xxx/CVE-2019-7306.json [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
+    ],
     "data_format": "MITRE",
     "data_type": "CVE",
     "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "Byobu Apport hook may disclose sensitive information since it automatically\nuploads the local user's .screenrc which may contain private hostnames,\nusernames and passwords.\nThis issue affects:\nbyobu"
```
Comment (iamleot) [inline 2019/7xxx/CVE-2019-7306.json:46]: Please mention fixed version also in the description.

Comment (setharnold) [reply 2019/7xxx/CVE-2019-7306.json:46]: It seems a helpful mergebot has already merged my branch -- do you have advice on how to proceed next? I'm relatively unskilled with github specifics. Shall I make a new branch for these two changes? Thanks

**Comment by iamleot** [comment]
>
> It seems a helpful mergebot has already merged my branch -- do you have advice on how to proceed next? I'm relatively unskilled with github specifics. Shall I make a new branch for these two changes? Thanks
>

Reopening a new pull request with these two changes should work.


Thanks!

**Pull request merged**